test/integration/transaction_test.exs

```diff
@@ -30,6 +30,60 @@ defmodule Braintree.Integration.TransactionTest do
     assert transaction.id =~ ~r/^\w+$/
   end
 
+  test "submit_for_settlement/2 can be used if transaction is authorized but not settling" do
+    {:ok, transaction} = Transaction.sale(%{
+      amount: "100.00",
+      payment_method_nonce: Nonces.paypal_one_time_payment
+    })
+    {:ok, settled_transaction} = Transaction.submit_for_settlement(transaction.id, %{})
+
+    assert settled_transaction.status == "settling"
+  end
+
+  test "submit_for_settlement/2 fails if transaction is already settling" do
+    {:ok, transaction} = Transaction.sale(%{
+      amount: "100.00",
+      payment_method_nonce: Nonces.paypal_one_time_payment,
+      options: %{submit_for_settlement: true}
+    })
+    {:error, error} = Transaction.submit_for_settlement(transaction.id, %{})
+
+    assert error.message == "Cannot submit for settlement unless status is authorized."
```

Comment:
Please match instead of asserting against the entire error message. Checking the full message is brittle (do we really care if they used a capital "C" in "Cannot" or had a trailing period?)